concourse/scripts/parse_buildpacks_and_run_uploader.rb

```diff
@@ -0,0 +1,8 @@
+yaml = YAML.safe_load(File.read("paas-cf/config/buildpacks.yml"))
```

Comment:
It would probably be better to pass in the path to the config file instead of hard-coding it here.